src/GamesList.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import 'bulma/css/bulma.min.css';
-import {Section, Box, Tile, Heading} from 'react-bulma-components';
+import { Section, Box, Tile, Heading } from 'react-bulma-components';
 import JoinGameButton from './JoinGameButton';
 
-const GamesList = ({game_sessions}) => {
+const GamesList = ({ game_sessions }) => {
   return (
     <Section>
       <Tile kind="ancestor" textAlign="center" vertical>
-        {game_sessions.map(gs => (
+        {game_sessions.filter(gs => gs.status !== "COMPLETED").map(gs => (
```

Comment:
@dinushkaherath this could get unwieldy down the road if there are hundreds of completed sessions in the database; I was hoping the filter could be on the query side

Reply:
switched to query side